providers/base/bin/cpuid.py

```diff
@@ -112,6 +112,7 @@
         "AMD ROME":         ['0x830f10'],
         "AMD Ryzen":        ['0x810f81'],
         "AMD Bergamo":      ['0xaa0f01'],
+        "AMD Hygon Dhyana"  ['0x900f22'],
```

Comment:
Don't call it an AMD, just "Hygon Dhyana Plus"
(I just looked at the info and apparently the 32 core is Dhyana Plus and there's a smaller 8-core thats just Dhyana).